dom-events/buttons.js

```diff
@@ -0,0 +1,27 @@
+function handleClick(event) {
+  console.log('button clicked');
+  console.log(event);
+  console.log(event.target);
+}
+
+var mouseEvent = document.querySelector('.click-button');
+
+mouseEvent.addEventListener('click', handleClick, Boolean);
```

Comment:
so when it says you can pass a `Boolean` value to it, it is talking about a `true` or `false` value. That is related to capturing and is not used often, so most of the time you can omit the third argument.

Reply:
makes sense!